mindsdb/interfaces/skills/sql_agent.py

```diff
@@ -251,11 +251,13 @@ def get_table_info(self, table_names: Optional[List[str]] = None) -> str:
         tables_info = []
         for table in all_tables:
             key = f"{ctx.company_id}_{table}_info"
-            table_info = self._cache.get(key) if self._cache else None
+            # Sanitize the key to avoid table (file) names with backticks and slashes.
+            sanitized_key = re.sub(r'[^\w\-.]', '_', key)
```

Comment:
As I understand the real problem is located [here](https://github.com/mindsdb/mindsdb/blob/4ea0d3f57fff448a9ce12155dd146c2b26027a1d/mindsdb/utilities/cache.py#L157) - file cache wrongly makes file name, it mustn't create subderictories if key name has `/`
Maybe do this update more globally - in fcache module?

Reply:
@ea-rus Are you suggesting that we add sanitization to the cache module? To replace slashes, back-ticks and the like there? Do you think there can be a situation where someone wants to place the file in an actual path (within directories) somewhere? I guess it is unlikely.

Reply:
@ea-rus I've moved the sanitization to the file cache.